util/testutil/testing.go

```diff
@@ -62,9 +63,16 @@ func NotOk(tb TB, err error, format string, a ...interface{}) {
 }
 
 // Equals fails the test if exp is not equal to act.
-func Equals(tb TB, exp, act interface{}) {
+func Equals(tb TB, exp, act interface{}, msgAndArgs ...interface{}) {
 	tb.Helper()
 	if !reflect.DeepEqual(exp, act) {
-		tb.Fatalf("\033[31m\nexp: %#v\n\ngot: %#v\033[39m\n", exp, act)
+		tb.Fatalf("\033[31m\nexp: %#v\n\ngot: %#v%s\033[39m\n", exp, act, formatMessage(msgAndArgs))
```

Comment:
I feel printing the message before `exp` and `got` would be better here.

Reply:
Added a commit to move the message first, similarly to tsdb